### PR TITLE
Repackage oauth2_proxy with rhel image

### PR DESCRIPTION
**Pull request by skabashnyuk**

### CODE REVIEW

File: build_deploy.sh [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+  tag_push ${REGISTRY}/openshiftio/rhche-che-oauth2_proxy:$TAG
+  tag_push ${REGISTRY}/openshiftio/rhche-che-oauth2_proxy:latest
+else
+  docker build -t rhche-che-oauth2_proxy -f "Dockerfile" .
+
+  tag_push ${REGISTRY}/openshiftio/rhche-che-oauth2_proxy:$TAG
+  tag_push ${REGISTRY}/openshiftio/frhche-che-oauth2_proxy:latest
+fi
+
+echo 'CICO: Image pushed, ready to update deployed app'
```
Comment (ibuziuk) [inline build_deploy.sh]: no new line

**Review by ibuziuk — APPROVED** [review]

File: build_deploy.sh [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+if [ "$TARGET" = "rhel" ]; then
+  docker build -t rhche-che-oauth2_proxy -f "Dockerfile.rhel" .
+
+  tag_push ${REGISTRY}/openshiftio/rhche-che-oauth2_proxy:$TAG
+  tag_push ${REGISTRY}/openshiftio/rhche-che-oauth2_proxy:latest
+else
+  docker build -t rhche-che-oauth2_proxy -f "Dockerfile" .
+
+  tag_push ${REGISTRY}/openshiftio/rhche-che-oauth2_proxy:$TAG
+  tag_push ${REGISTRY}/openshiftio/frhche-che-oauth2_proxy:latest
```
Comment (aditya-konarde) [inline build_deploy.sh]: There's an extra `f` in the name of the image here

**Review by aditya-konarde — CHANGES_REQUESTED** [review]

File: build_deploy.sh (hunk 2 above)
Comment (aditya-konarde) [reply build_deploy.sh]: `frhche` should be `rhche`

**Review by aditya-konarde — APPROVED** [review]
LGTM

**Pull request merged**